baselines/flwr_baselines/publications/fedprox_mnist/tests.py

```diff
@@ -0,0 +1,35 @@
+"""Used to test the model and the data partitionning."""
```

Comment:
The entire module `tests.py` should be renamed to belong to one of the other modules, for example, `model_test.py`. If functions from multiple modules are being tested (e.g., `model` and `dataset`), then we should split this up into multiple test modules, each belonging to one of the existing non-test modules